capstone/src/main/resources/static/manageevents.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta>
+    <title>My Account</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="user.js"></script>
+</head>
+<body onload="getUser(false);getOrganizationEvents();">
+    <nav class="nav-bar">
+        <ul>
+            <li class="home"><a href="">College Activity Finder</a></li>
+            <li><a href="index.html">Events</a></li>
+            <li><a href="organizationsearch.html">Organizations</a></li>
+            <li class="active"><a href="profile.html">My Account</a></li>
+            <li><a href="">About Us</a></li>
+        </ul>
+    </nav>
+    <br/>
+    <div class="profile-head">
+        <div id="name"></div>
+        <div class="container">
+            <a href="images/js7.jpg"><img class="image profile" src="images/js7.jpg"/></a>
+        </div>
+        <div id="email"></div>
+        <div id="university"></div>
+        <div id="description"></div>
+        <div id="no-profile">
+            <p>You have not built your profile yet, please submit a profile </p>
+        </div>
+        <nav class="nav-bar-inverted" id="individual-nav">
```

Comment:
Should this be org-nav? Since manageevents.html seems like a org profile page?

Reply:
No it should be individual nav. Each of the pages has two nav bars that gets hidden or displayed based on user type. The second nav is right below this one.